recipes/coin3d/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "coin3d" %}
+{% set version = "4.0.0" %}
+{% set sha256 = "8319cfa699afac0e5f19054222c25a7ad655f018f7c3c7b53f626e7b167cb08e" %}
+{% set download_url = "https://github.com/FreeCAD/pivy/releases/download/0.6.3/Coin3D-coin-8e8337f61b05.tar.gz" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}.tar.gz
+    url: {{ download_url }}
+    sha256: {{ sha256 }}
+
+    patches:
+        - include.patch
```

Comment:
Add `toolchain` to build requirement and remove this patch

Reply:
it seems to me like this doesn't work for osx. 
https://travis-ci.org/conda-forge/staged-recipes/builds/326143247#L402
```
[  0%] Building CXX object src/soscxml/CMakeFiles/soscxml.dir/ScXMLCoinEvaluator.cpp.o
/Users/travis/miniconda/conda-bld/coin3d_1515358497548/work/Coin3D-coin-8e8337f61b05/src/xml/document.cpp:45:10: fatal error: 'boost/scoped_array.hpp' file not found
#include <boost/scoped_array.hpp>
         ^
1 error generated.
```